apps/studio/src/utils/resources.ts

```diff
@@ -19,3 +50,14 @@ export const isAllowedToHaveLastEditedText = (
     resourceType === ResourceType.CollectionPage
   )
 }
+
+// only show user-viewable resources (excluding root page, folder meta etc.)
+export const getUserViewableResourceTypes = (): ResourceType[] => {
```

Comment:
nit: we probably should make a custom type like `UserViewableResourceType = Extract...`

Reply:
right now we are using it more of an array of values (const) rather than a type, so should be okay to leave it as it is